sql/core/src/test/scala/org/apache/spark/sql/execution/command/PlanResolutionSuite.scala

```diff
@@ -143,7 +149,8 @@ class PlanResolutionSuite extends AnalysisTest {
       new ResolveCatalogs(catalogManager),
       new ResolveSessionCatalog(catalogManager, conf, _ == Seq("v")),
       analyzer.ResolveTables,
-      analyzer.ResolveRelations)
+      analyzer.ResolveRelations,
+      new ResolveSessionCatalog(catalogManager, conf, _ == Seq("v")))
```

Comment:
We will need to run this again until we move all commands to the new resolution framework.